disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +204,17 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
+    """
+    This function takes as input an integer that represents a cycle count,
+    calculates jitter and executes sleep for the calculated time.
+    The value of 'cycle' must be an integer greater than 0.
+    """
+    base = 3
+    if cycle is None or cycle < 1:
+        raise ValueError('Value of cycle must be int > 0')
+
+    time.sleep(min(MAX_POLL_INTERVAL, randint(base, cycle * 3)))
```

Comment:
I think there could be a better algoritham here for jitter. My concern is that it seems likely that this will tend to end up with values that are mostly `60` because of the `min(MAX_POLL_INTERVAL, ...)`. What about something like:
```python
jitter = randint(-3, 3)
seconds_to_sleep = base + math.pow(2, cycle)
jittered_seconds_to_sleep = min(MAX_POLL_INTERVAL, seconds_to_sleep) + jitter
time.sleep(jittered_seconds_to_sleep)
```

Thoughts?